posthog/settings/feature_flags.py

```diff
@@ -8,6 +8,4 @@
     "invite-teammates-prompt",
     "insight-legends",
     "experiments-secondary-metrics",
-    "8545-ff-activity-log",
```

Comment:
This one is still present in `activityLogMocks.ts`, since the flag is no longer needed can `featureFlagsActivityResponseJson` be eliminated in that file?

Reply:
Ah, it was examples for the storybook page for activity log

But also highlighted that the story was broken. Updated the examples and fixed the page